what4/CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# next
+
+* The `BoolMap` parameter of `ConjPred` is now a `ConjMap`.
```

Comment:
`ConjMap` is a new addition to this version of `what4`, so it would be worth adding a brief explanation of what `ConjMap` is as well.